--- pcre-7.7/configure.ac.orig	2008-05-07 18:22:03.000000000 +0200
+++ pcre-7.7/configure.ac	2008-06-17 19:40:42.879677506 +0200
@@ -533,6 +533,8 @@
 AC_SUBST(EXTRA_LIBPCRE_LDFLAGS)
 AC_SUBST(EXTRA_LIBPCREPOSIX_LDFLAGS)
 AC_SUBST(EXTRA_LIBPCRECPP_LDFLAGS)
+AC_SUBST(PCREGREP_LIBS)
+AC_SUBST(PCRETEST_LIBS)
 
 # When we run 'make distcheck', use these arguments.
 DISTCHECK_CONFIGURE_FLAGS="--enable-cpp --enable-unicode-properties"
@@ -550,7 +552,7 @@
     echo "** Cannot --enable-pcregrep-libz because libz was not found"
     exit 1
   fi
-  if test "$LIBS" = ""; then LIBS=-lz; else LIBS="$LIBS -lz"; fi
+  if test "$PCREGREP_LIBS" = ""; then PCREGREP_LIBS=-lz; else PCREGREP_LIBS="$PCREGREP_LIBS -lz"; fi
 fi
 
 if test "$enable_pcregrep_libbz2" = "yes"; then
@@ -562,7 +564,7 @@
     echo "** Cannot --enable-pcregrep-libbz2 because libbz2 was not found"
     exit 1
   fi
-  if test "$LIBS" = ""; then LIBS=-lbz2; else LIBS="$LIBS -lbz2"; fi
+  if test "$PCREGREP_LIBS" = ""; then PCREGREP_LIBS=-lbz2; else PCREGREP_LIBS="$PCREGREP_LIBS -lbz2"; fi
 fi
 
 # Similarly for --enable-pcretest-readline
@@ -576,7 +578,7 @@
     echo "** Cannot --enable-pcretest-readline because readline/history.h was not found."
     exit 1
   fi
-  if test "$LIBS" = ""; then LIBS=-lreadline; else LIBS="$LIBS -lreadline"; fi
+  if test "$PCRETEST_LIBS" = ""; then PCRETEST_LIBS=-lreadline; else PCRETEST_LIBS="$PCRETEST_LIBS -lreadline"; fi
 fi
 
 # Produce these files, in addition to config.h.
--- pcre-7.7/Makefile.am.orig	2008-01-23 18:58:28.000000000 +0100
+++ pcre-7.7/Makefile.am	2008-06-17 19:43:11.376139837 +0200
@@ -245,13 +245,13 @@
 EXTRA_DIST += RunTest.bat
 bin_PROGRAMS += pcretest
 pcretest_SOURCES = pcretest.c
-pcretest_LDADD = libpcreposix.la
+pcretest_LDADD = libpcreposix.la $(PCRETEST_LIBS)
 
 TESTS += RunGrepTest
 dist_noinst_SCRIPTS += RunGrepTest
 bin_PROGRAMS += pcregrep
 pcregrep_SOURCES = pcregrep.c
-pcregrep_LDADD = libpcreposix.la
+pcregrep_LDADD = libpcreposix.la $(PCREGREP_LIBS)
 
 EXTRA_DIST += \
   testdata/grepinput \
