--- pcre-6.1/configure.in~	2005-06-22 16:58:14.218414248 +0200
+++ pcre-6.1/configure.in	2005-06-22 16:58:37.476878424 +0200
@@ -96,7 +96,7 @@
 AC_LANG_RESTORE
 
 dnl From the above, we now have enough info to know if C++ is fully installed
-if test -n "$ac_cv_prog_ac_ct_CXX" -a "$pcre_has_cpp_headers" = 1; then
+if test -n "$CXX" -a "$pcre_has_cpp_headers" = 1; then
   MAYBE_CPP_TARGETS='$(CPP_TARGETS)'
 fi
 AC_SUBST(MAYBE_CPP_TARGETS)
